codespell_lib/data/dictionary.txt

```diff
@@ -7597,12 +7602,16 @@ filetests->file tests
 fileystem->filesystem
 fileystems->filesystems
 filiament->filament
+filing->filling
+fillay->fillet
 fille->file, fill, filled,
+fillement->filament
 filles->files, fills, filled,
 fillowing->following
 fillung->filling
 filnal->final
 filname->filename
+filo->phyllo
```

Comment:
Remove, style, also the same word: https://en.wikipedia.org/wiki/Filo

Reply:
delete